modules/lang-mustache/src/yamlRestTest/resources/rest-api-spec/test/lang_mustache/30_search_template.yml

```diff
@@ -140,6 +140,60 @@
   - length: { profile: 1 }
 
 ---
+"Explain as query string":
```

Comment:
This is certainly a good way to test, but I would also like to see a unit test for instance as part of RestSearchTemplateActionTests